Quadriga/src/test/java/edu/asu/spring/quadriga/service/conceptcollection/mapper/impl/ConceptCollectionDeepMapperTest.java

```diff
@@ -0,0 +1,255 @@
+import edu.asu.spring.quadriga.dto.ConceptCollectionItemsDTOPK;
+import edu.asu.spring.quadriga.dto.ConceptsDTO;
+import edu.asu.spring.quadriga.dto.QuadrigaUserDTO;
+import edu.asu.spring.quadriga.exceptions.QuadrigaStorageException;
+import edu.asu.spring.quadriga.service.IQuadrigaRoleManager;
+import edu.asu.spring.quadriga.service.user.mapper.IUserDeepMapper;
+import edu.asu.spring.quadriga.service.workbench.mapper.IProjectConceptCollectionShallowMapper;
+import edu.asu.spring.quadriga.service.workspace.mapper.IWorkspaceCCShallowMapper;
+
+public class ConceptCollectionDeepMapperTest {
+
+	@Mock
+	private IConceptCollectionDAO mockeddbConnect = Mockito.mock(IConceptCollectionDAO.class);
+
+	@Mock
+	private IConceptCollectionFactory mockedccFactory = Mockito.mock(IConceptCollectionFactory.class);
+
+	@Mock
+	private IUserDeepMapper mockedUserDeepMapper = Mockito.mock(IUserDeepMapper.class);
+
+	@Mock
+	private IProjectConceptCollectionShallowMapper mockedProjectCCShallowMapper = Mockito
+			.mock(IProjectConceptCollectionShallowMapper.class);
+
+	@Mock
+	private IWorkspaceCCShallowMapper mockedWorkspaceCCShallowMapper = Mockito.mock(IWorkspaceCCShallowMapper.class);
+
+	@Mock
+	private IQuadrigaRoleManager mockedRoleManager = Mockito.mock(IQuadrigaRoleManager.class);
+
+	@Mock
+	private ICollaboratorFactory mockedCollaboratorFactory = Mockito.mock(ICollaboratorFactory.class);
+
+	@Mock
+	private IConceptCollectionCollaboratorFactory mockedccCollaboratorFactory = Mockito
+			.mock(IConceptCollectionCollaboratorFactory.class);
+
+	@Mock
+	private IConceptFactory mockedConceptFactory = Mockito.mock(IConceptFactory.class);
+
+	@Mock
+	private IConceptCollectionConceptFactory mockedccConceptsFactory = Mockito
+			.mock(IConceptCollectionConceptFactory.class);
+
+	@InjectMocks
+	ConceptCollectionDeepMapper conceptCollectionDeepMapperUnderTest;
+
+	@Before
+	public void setUp() {
+		MockitoAnnotations.initMocks(this);
+	}
+
+	@Test
+	public void getConceptCollectionDetailsTest() throws QuadrigaStorageException {
+
+		ConceptsDTO conceptDTO = new ConceptsDTO();
+		conceptDTO.setLemma("lemma");
+
+		ConceptCollectionItemsDTOPK conceptCollectionItemsDTOPK = new ConceptCollectionItemsDTOPK();
+		conceptCollectionItemsDTOPK.setConcept("concept");
+
+		ConceptCollectionItemsDTO ccItemsDTO = new ConceptCollectionItemsDTO();
+		ccItemsDTO.setConceptCollectionItemsDTOPK(conceptCollectionItemsDTOPK);
+
+		ConceptCollectionItemsDTO collectionItemsDTO = new ConceptCollectionItemsDTO();
+		collectionItemsDTO.setConceptcollectionsItemsDTOPK(conceptCollectionItemsDTOPK);
+		collectionItemsDTO.setConceptDTO(conceptDTO);
+		ccItemsDTO.setConceptcollectionsItemsDTOPK(conceptCollectionItemsDTOPK);
+
+		List<ConceptCollectionItemsDTO> conceptCollectionItemsDTOList = new ArrayList<ConceptCollectionItemsDTO>();
+		conceptCollectionItemsDTOList.add(collectionItemsDTO);
+
+		ConceptCollectionDTO ccDTO = createTestConceptCollectionDTO();
```

Comment:
Why don't you create a setup method and initialize a conceptcollectiondto there?